locales/ca.json

```
@@ -457,21 +601,32 @@
   },
   "state": {
     "attachments_exceed_server_limit": "El nombre de fitxers adjunts ha superat el límit per missatge.",
+    "attachments_limit_audio_error": "S'ha superat la mida màxima d'àudio: {0}",
     "attachments_limit_error": "S'ha superat el límit per publicació",
+    "attachments_limit_image_error": "S'ha superat la mida màxima d'imatge: {0}",
+    "attachments_limit_unknown_error": "S'ha superat la mida màxima de fitxer: {0}",
+    "attachments_limit_video_error": "S'ha superat la mida màxima de vídeo: {0}",
     "edited": "(Edit)",
     "editing": "Edició",
     "loading": "Carregant...",
     "publish_failed": "No s'ha pogut publicar",
     "publishing": "Publicació",
+    "save_failed": "Error de desar",
     "upload_failed": "La càrrega ha fallat",
     "uploading": "S'està carregant..."
   },
   "status": {
+    "account": {
+      "suspended_message": "El compte d'aquest missatge ha estat suspès.",
+      "suspended_show": "Mostrar contingut igualment?",
```

The CI error seems to be caused by just an extra `,` character. 🙂 

```suggestion
      "suspended_show": "Mostrar contingut igualment?"
```